packages/web-components/fast-foundation/src/number-field/number-field.ts

```diff
@@ -261,6 +261,24 @@ export class NumberField extends FormAssociatedNumberField {
     public handleChange(): void {
         this.$emit("change");
     }
+
+    /**
+     * Handles the internal control's `keydown` event
+     * @internal
+     */
+    public handleKeyDown(event: KeyboardEvent): boolean {
+        if (event.key === "ArrowUp") {
```

Comment:
You should include the arrow key codes and check using those.

```
import { keyCodeArrowDown, keyCodeArrowUp } from "@microsoft/fast-web-utilities";
```

Then check the keycode:

```
if (event.keyCode === keyCodeArrowUp) {
    ...
}
```

Reply:
As [`KeyboardEvent.keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), I opted to use a [widely supported syntax](https://caniuse.com/keyboardevent-key) to achieve the same thing without importing an additional utility.

May I ask why we're still using key codes now that the browsers bake more intuitive values into `KeyboardEvent.key`?

I'm happy to make the change if that's what it takes — just seems like a step backwards.

Reply:
> As [`KeyboardEvent.keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), I opted to use a [widely supported syntax](https://caniuse.com/keyboardevent-key) to achieve the same thing without importing an additional utility.

This is correct, we should moving away from keyCode.